packages/sources/coingecko-test/src/endpoint/global-marketcap.ts

```diff
@@ -1,25 +1,20 @@
-import { HttpRequestConfig, HttpResponse } from '@chainlink/external-adapter-framework/transports'
 import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { HttpRequestConfig } from '@chainlink/external-adapter-framework/transports'
 import { BatchWarmingTransport } from '@chainlink/external-adapter-framework/transports/batch-warming'
 import { ProviderResult } from '@chainlink/external-adapter-framework/util'
 import {
   buildGlobalRequestBody,
-  AdapterRequestParams,
-  ProviderResponseBody,
   constructEntry,
+  GlobalEndpointTypes,
   inputParameters,
 } from '../global-utils'
-import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
 
-const batchEndpointTransport = new BatchWarmingTransport({
-  prepareRequest: (_: AdapterRequestParams[], config: AdapterConfig): HttpRequestConfig<never> => {
+const batchEndpointTransport = new BatchWarmingTransport<GlobalEndpointTypes>({
+  prepareRequest: (_, config): HttpRequestConfig<never> => {
     return buildGlobalRequestBody(config.API_KEY)
   },
-  parseResponse: (
-    params: AdapterRequestParams[],
-    res: HttpResponse<ProviderResponseBody>,
-  ): ProviderResult<AdapterRequestParams>[] => {
-    const entries = [] as ProviderResult<AdapterRequestParams>[]
+  parseResponse: (params, res): ProviderResult<GlobalEndpointTypes>[] => {
```

Comment:
I thought after the types changes we wouldn't have to define the return type like the crypto endpoints?

Reply:
We don't have to necessarily, it's just convenient sometimes haha